src/pages/graphql/reference/legacy/index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't think "legacy" is quite right. I think using the version number would be cleaner.